01_install_requirements.sh

```diff
@@ -59,3 +49,51 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
   -e "virthost=$HOSTNAME" \
   -i vm-setup/inventory.ini \
   -b -vvv vm-setup/install-package-playbook.yml
+
+# Create Go directory
+eval $(go env)
+mkdir -p ${GOPATH}/{bin,src,pkg/mod} 
+
+if [ $K8S == "minikube" ]; then
+  if ! which minikube 2>/dev/null ; then
+      curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+      chmod +x minikube
+      sudo mv minikube /usr/local/bin/.
+  fi
+  
+  if ! which docker-machine-driver-kvm2 2>/dev/null ; then
+      curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2
+      chmod +x docker-machine-driver-kvm2
+      sudo mv docker-machine-driver-kvm2 /usr/local/bin/.
+  fi
+else
+# Install kinder to replace minikube
+  if [ ! -d ${GOPATH}/src/kubeadm ]; then
+    pushd ${GOPATH}/src/
+    git clone https://github.com/kubernetes/kubeadm.git
+    pushd kubeadm
+    popd
+    popd
+  fi
+  
+  # if [ ! -d ${GOPATH}/src/sigs.k8s.io/kind ]; then
+  #   GO111MODULE="on" go get -u sigs.k8s.io/kind@v0.4.0
+  # fi
```

Comment:
want to remove this?  or add a comment about why it's here?

Reply:
removed it!